gcp_census/bigquery/transformers/table_metadata_v1_0.py

```diff
@@ -22,14 +22,28 @@ def transform(self):
             'numBytes': table['numBytes'],
             'numLongTermBytes': table['numLongTermBytes'],
             'numRows': table['numRows'],
-            'timePartitioning': table[
-                'timePartitioning'] if 'timePartitioning'
-                                       in table else None,
+            'timePartitioning': self.__copy_time_partitioning(table),
             'type': table['type'],
             'json': json.dumps(table),
             'description': table['description']
             if 'description' in table else None,
-            'labels': [{'key': key, 'value': value} for key, value
-                       in table['labels'].iteritems()]
-            if 'labels' in table else []
+            'labels': self.__copy_labels(table)
         }
+
+    @staticmethod
+    def __copy_labels(table):
```

Comment:
It looks like code duplication, could you extract it to some common source for both?

Reply:
table_metadata_v0_1.py is already deprecated and should be removed (after fixing few other issues), so I'm reluctant to create something which will be removed in medium term